src/beacon-api/app/services/device_service.py

```
@@ -949,135 +907,141 @@ async def sync_device_sites(db: Session, token: str) -> Dict[str, Any]:
-                        db_cat.description = cat.get("description")
-                        cat_updated = True
-                    if cat_updated:
-                        db.add(db_cat)
-                
-                categories_synced.add(cat_name)
-            
-            # Commit in batches of 50 to avoid long transactions
             if (updates_count + new_count) % 50 == 0:
                 db.commit()
-                
         except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)